joomla/joomla-cms
# [4.0][CLI] version option

### Summary of Changes
~~added a CLI command to retrieve the installed version of the Joomla CMS~~

Added an option -V  or --version to display the CMS version


### Testing Instructions
make sure you have this one  joomla-framework/console@6727b2b
apply the patch go to the terminal
run  `php cli\joomla.php`
you'll see a new option

~~you'll see a new entry on the available cli commands `core:version`~~
![Screenshot from 2020-07-18 08-35-13](https://user-images.githubusercontent.com/181681/87846651-b5adae80-c8d1-11ea-9fbc-ebb482537be5.png)




### Actual result BEFORE applying this Pull Request
~~no core:version command listed~~


### Expected result AFTER applying this Pull Request
~~core:version command is listed~~ 
and if you run 
`php cli/joomla.php -V
`
this is the result 
![Screenshot from 2020-07-18 08-41-05](https://user-images.githubusercontent.com/181681/87846751-7df33680-c8d2-11ea-9c61-3b623459d493.png)



### Documentation Changes Required
yes
